ckanext/scheming/commands.py

```diff
@@ -25,41 +23,17 @@ class SchemingCommand(CkanCommand):
 
     parser = paste.script.command.Command.standard_parser(verbose=True)
     parser.add_option('-c', '--config', dest='config',
-                      default='development.ini',
+                      default=os.getenv('CKAN_INI', 'development.ini'),
                       help='Config file to use.')
 
     def command(self):
-        cmd = self.args[0]
         self._load_config()
+        if not self.args:
+            print(self.__doc__)
+            return
 
+        cmd = self.args[0]
         if cmd == 'show':
-            self._show()
+            print(utils.describe_schemas())
         else:
-            print self.__doc__
-
-    def _show(self):
-        schemas = [
-            ("Dataset", scheming_dataset_schemas()),
-            ("Group", scheming_group_schemas()),
-            ("Organization", scheming_organization_schemas()),
-        ]
-
-        for n, s in schemas:
-            print n, "schemas:"
-            if s is None:
-                print "    plugin not loaded or schema not specified\n"
-                continue
-            if not s:
-                print "    no schemas"
-            for typ in sorted(s):
-                print " * " + json.dumps(typ)
-                field_names = ('dataset_fields', 'fields', 'resource_fields')
-
-                for field_name in field_names:
-                    if s[typ].get(field_name):
-                        if field_name == 'resource_fields':
-                            print " * " + json.dumps("resource")
-                        for field in s[typ][field_name]:
-                            print "   - " + json.dumps(field['field_name']),
-                            print scheming_language_text(field.get('label'))
-            print
+            print(self.__doc__)
```

Comment:
I'm fine with removing the scheming cli instead; it does nothing that you can't already do with the API now.

Reply:
The less code, the better